rpi_sensors_exporter/README.md

```diff
@@ -0,0 +1,5 @@
+## Development notes
+Adding support for new sensor:
+* In metrics.py add the necessary metrics depending on the data provided by the sensor, considering metric and label naming best practices - https://prometheus.io/docs/practices/naming/
+* Add "Metrics" class in sensors dir, which gets the sensor data and populates the metrics
+* Add the corresponding sensor in exporter.py. Keep in mind that depending on the underlying library for the specific sensor a different exception will be produced if a sensor is not connected.
```

Comment:
Fix this sentence:
   * Keep in mind that depending on the underlying library for the specific sensor a different exception will be produced if a sensor is not connected.
For example:
  * Keep in mind that depending on underlying library for your sensor. It is possible that exception different than the expected are thrown.